src/core/game.ts

```diff
@@ -96,25 +96,44 @@ export class Game {
       Object.assign(list, (game.entities as any)[key]);
       return list;
     }, {});
+    let { walls, players }  = game.entities;
 
     events.forEach(event => {
       let sender = allEntities[event.initiator];
       let receiver = allEntities[event.receptor];
       switch(event.type) {
-        case 'COLLISION':
+        case 'COLLISION': // handles static motion
           switch(sender.type) {
             case 'player': Player.collideWith(sender as PlayerState, receiver, game); break;
             case 'bullet': Bullet.collideWith(sender as BulletState, receiver, game); break;
           }
           break;
-        case 'SPAWN_BULLET':
+        case 'SPAWN_BULLET': // spawns the missiles
```

Comment:
Static: not as a result of player input, once set doesn't change
Dynamic: Motion could change at any time.